src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -789,6 +789,21 @@ bool CompilerOpenFPGA::DesignChanged(
   return result;
 }
 
+std::vector<std::string> CompilerOpenFPGA::GetFiles(
+    Action action, const std::string& projectName) const {
+  std::vector<std::string> files;
+  switch (action) {
+    case Compiler::Action::Analyze:
+      files = {ANALYSIS_LOG, "port_info.json",
```

Comment:
Did you look into diff in the folder location for files generated in-between the tasks ?

Reply:
I didn't follow. Could you please elaborate with an e.g. ?